tutorials/getting-started/custom-widgets.md

```diff
@@ -103,6 +103,8 @@ Let's make another module and its views folder in one step:
 mkdir -p lib/modules/page-link-widgets/views
 ```
 
+>While it's good to get some experience making all the folders and files yourself, the `apostrophe-cli` pacakage that you installed earlier can easily create basic module structures for you from templates. You can learn more in the [`apostrophe-cli` README](https://github.com/apostrophecms/apostrophe-cli/blob/master/README.md).
```

Comment:
We need a space after the `>` for blockquote (right?) or go with the hint block for Gitbook.

Reply:
Yes - good catch. I'm still getting used to Gitbook syntax. I'm going to go back through and hit all of the blockquotes after this with the correct styles, so we can just leave this as is for now.